menu-definitions.lisp

```diff
@@ -32,6 +32,9 @@
           select-from-batch-menu
           command-menu))
 
+(defun car-safe (lst)
+  (if (consp lst) (car lst) lst))
+
```

Comment:
Please do not use car-safe.  There's a reason why this isn't built into the language.